sct.py

```diff
@@ -1594,6 +1596,25 @@ def generate_parallel_timelines_report(logdir: str | None, test_id: str | None)
     pt_report_generator.generate_full_report()
 
 
+@cli.command("create-es-index", help="Create ElasticSearch index with mapping ")
+@click.option("-n", "--name", envvar='SCT_ES_INDEX_NAME', help="ES index name")
+@click.option("-dt", "--doc-type", envvar='SCT_ES_DOC_TYPE', default="")
+@click.option("-f", "--mapping-file", envvar='SCT_MAPPING_FILEPATH', type=click.Path(exists=True),
+              help="Full path to es index mapping file")
+def create_es_index(name: str, doc_type: str, mapping_file: str) -> None:
```

Comment:
Can you elaborate somewhere like in `/docs` about the process in which someone would use this command ?

Reply:
Added docs/hydra-create-es-index.md